internal/pkg/manifest/validate_env.go

```diff
@@ -290,6 +290,12 @@ func (cfg EnvironmentHTTPConfig) validate() error {
 
 // validate returns nil if PublicHTTPConfig is configured correctly.
 func (cfg PublicHTTPConfig) validate() error {
+	if !cfg.SecurityGroupConfig.Ingress.IsEmpty() && !cfg.Ingress.IsEmpty() {
```

Comment:
To elaborate a bit for fellow reviewer's thoughts  <img src="https://slackmojis.com/emojis/12806-meow_attention/download" width="25px"/>:

`cfg.SecurityGroupConfig.Ingress` is the existing field that has been there in multiple releases, `cfg.Ingress` is the new field that fixes it (in one of the upcoming release).

Since it's a new field - if it's non-empty, it must mean that the user has modified their manifest, adding configurations under `http.public.ingress`. Erroring out in this case is not "breaking".

On the other hand, manifests with `http.public.security_groups.ingress` field configured will keep working - given that they have not been modified to use `http.public.ingress`.